changelog/8838.removal.rst

```diff
@@ -0,0 +1,17 @@
+As per our policy, the following features have been deprecated in the 6.X series and are now
```

Comment:
This file should be named `8838.breaking.rst`.

Reply:
Right, fixed.